generator/spring-boot/src/main/java/io/fabric8/maven/generator/springboot/SpringBootGenerator.java

```diff
@@ -84,6 +89,12 @@ public boolean isApplicable(List<ImageConfiguration> configs) {
         if (Boolean.parseBoolean(getConfig(color))) {
             opts.add("-Dspring.output.ansi.enabled=" + getConfig(color));
         }
+
+        //Spring boot active profiles
+        String strActiveProfiles  = getConfig(activeProfiles);
+        if(strActiveProfiles!=null) {
+            opts.add("-Dspring.profiles.active="+strActiveProfiles);
```

Comment:
quoted it - but we need to see how it works hen deployed in k8s, sometimes i see it does not work as expected ..